app/ItemList.java

```diff
@@ -0,0 +1,47 @@
+package icebase.app;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+import icebase.icebase.Icebase;
+
+public class ItemList {
+    ItemList(){
+
+    }
+
+    public static void buyItem(String item){
+        System.out.println(item);
+        //display other item attributes
+    }
+    public static void main(String[] args){
+        Icebase ib = new Icebase();
+        Scanner sc = new Scanner(System.in);
+        int choice;
+        String storeName = "STORE X";
+        String categoryName = "CATEGORY X";
+        String[] itemList = {"Item A", "Item B", "Item C"}; //fetch from icebase
+        int range = itemList.length;
+        while (true){
+            System.out.printf("\t\t\t ---------%n\t\t\t| %s |%n\t\t\t ---------%n", storeName);
+            MenuTitle.displaySubTitle("[" + categoryName + "]");
+            for(int i = 0; i < range; i++){
+                System.out.printf(
+                                    "\t\t[%d] - %s%n", i+1, itemList[i]);
+            }
+            System.out.printf("\t\t[%d] - Return%n", range+1);
+
+            System.out.print("\nChoice: ");
+            try{
+                choice = Integer.parseInt(sc.nextLine());
+                if(choice == range+1) break;
```

Comment:
put parenthesis

Reply:
nya need pod siguro checker for the inputs here, like if negative, etc etc

Reply:
ang next line is i use man for index so muerror ang negatives and out of range nga inputs, also for float and alnum

Reply:
AYY YAWA HAHAHA geenric Exception man diay gigamit oks oks